taxcalc/taxcalcio.py

```diff
@@ -47,11 +47,11 @@ class TaxCalcIO(object):
         or string is name of optional ASSUMP file.
 
     growdiff_response: Growdiff object or None
-        growdiff_response Growdiff object used in dynamic analysis;
-        must be None when conducting static analysis.
+        growdiff_response Growdiff object used only in dynamic analysis;
+        must be None when not conducting dynamic analysis.
```

Comment:
Not sure whether it is worth splitting hairs here in the code, but growdiff_response must also be None when conducting a partial-equilibrium dynamic analysis.